source/code-generation/query-converter/manage-queries.txt

```diff
@@ -0,0 +1,41 @@
+.. _rm-manage-queries:
+
+==============
+Manage Queries
+==============
+
+.. contents:: On this page
```

Comment:
(info typing) 
Ooof. I'm sorry to say, I think each of these actions might need to be it's own page. I understand and appreciate the impulse to make this one page, but I'm not sure it works. Let me know what you think and we can totes take another look once you've got all the actions written up.

Reply:
Happy to break this out into pages. We can discuss further though once the steps are developed (I can see this getting long now that it's filling int). 

Sigh, the eternal IA struggle between info typing correctly and ToC bloat.